src/main/kotlin/com/example/minibank/controllers/AuthController.kt

```diff
@@ -117,11 +162,16 @@
     @PostMapping("/logout")
     @Operation(summary = "Деавторизация пользователя")
     fun logout(response: HttpServletResponse): ResponseEntity<Any> {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty